catalog/transformer/catalog-transformer-zip/src/main/java/org/codice/ddf/catalog/transformer/zip/ZipCompression.java

```diff
@@ -46,7 +46,7 @@
 import ddf.catalog.operation.ResourceRequest;
 import ddf.catalog.operation.ResourceResponse;
 import ddf.catalog.operation.SourceResponse;
-import ddf.catalog.operation.impl.ResourceRequestByProductUri;
+import ddf.catalog.operation.impl.ResourceRequestById;
```

Comment:
@bdeining just wanted you to be aware this part is a backport of https://github.com/codice/ddf/pull/945